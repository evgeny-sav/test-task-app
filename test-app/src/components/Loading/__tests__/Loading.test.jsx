import React from 'react';
import { create } from 'react-test-renderer';
import Loading from '../Loading';

describe('Loading', () => {
  it('should render', () => {
    const component = create(<Loading />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
