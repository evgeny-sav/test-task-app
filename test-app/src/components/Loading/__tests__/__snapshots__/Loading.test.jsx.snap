// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders without crashing 1`] = `
<div
  className="Loading"
>
  <div
    className="lds-roller"
  >
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;

exports[`Loading should render 1`] = `
<div
  className="Loading"
>
  <div
    className="lds-roller"
  >
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;
